src/mixxxapplication.cpp

```diff
@@ -65,6 +66,8 @@ void MixxxApplication::registerMetaTypes() {
     qRegisterMetaType<mixxx::ReplayGain>("mixxx::ReplayGain");
     qRegisterMetaType<mixxx::Bpm>("mixxx::Bpm");
     qRegisterMetaType<mixxx::Duration>("mixxx::Duration");
+    qRegisterMetaType<SoundDeviceId>("SoundDeviceId");
```

Comment:
The string argument is not needed if symbols are part of the global namespace

Reply:
Yes, I thought it was good to include it for consistency with the surrounding code.